cmd/lncli/commands.go

```diff
@@ -3380,27 +3385,30 @@ func forwardingHistory(ctx *cli.Context) error {
 		err                    error
 	)
 	args := ctx.Args()
+	now := time.Now()
 
 	switch {
 	case ctx.IsSet("start_time"):
-		startTime = ctx.Uint64("start_time")
+		startTime, err = parseTime(ctx.String("start_time"), now)
 	case args.Present():
-		startTime, err = strconv.ParseUint(args.First(), 10, 64)
-		if err != nil {
-			return fmt.Errorf("unable to decode start_time %v", err)
-		}
+		startTime, err = parseTime(args.First(), now)
```

Comment:
Default value?

Reply:
I rebased and had to adjust the command description further above a little, else no obvious conflicts.

Reply:
Yeah, I was confused about where the default case on this switch was, but it's below now because of the rebase.